.github/workflows/production.yml

```diff
@@ -0,0 +1,37 @@
+name: AWS Build and Deploy
+on:
+  pull_request:
```

Comment:
be careful with this. We want to test the test the UAT on PR scrpt on pull_request to master. Otherwise push to master would be the choice.